This fixes an issue where Musl doesn't set itself as POSIX compatible

diff --git a/meson.build b/meson.build
index 567ee7a..6bbbe28 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,7 @@
 project('gxr', 'c', version: '0.15.1',
         meson_version: '>= 0.49.0',
         default_options : [
-          'c_std=c11',
+          'c_std=gnu99',
           'warning_level=3',
         ],
 )
